guest-tools/direct-boot/README.md

```diff
@@ -0,0 +1,140 @@
+# TD Boot Methods 
+
+This folder contains scripts and instructions to boot a TD using the [direct
+boot](https://qemu-project.gitlab.io/qemu/system/linuxboot.html) method.
+
+The direct boot method is an alternative to the boot method that is being used by `tdvirsh`
+to run TDs.
+We will refer to the `tdvirsh` boot method as the `indirect boot` method.
+With the `indirect boot` method, the boot chain involves following components:
+- TDVF (virtual UEFI firmware)
+- SHIM
+- Grub
+- Kernel + Initrd
+
+The `direct boot` will skip `SHIM` and `Grub` in the boot chain by providing the `Kernel`
+and `Initrd` directly to `qemu`. Per consequence, the boot chain involves these components:
+- TDVF (virtual UEFI firmware)
+- Kernel + Initrd
+
+The boot chain structure is important for remote attestation since it impacts the size of
+the event log journal. Indeed, each component in the boot chain generates a set of entries of the event
+log journal. The more components we have in the boot chain, the more event logs we will have and the harder
+is the verification of the correctness of the measurement values.

```

Comment:
This is also a bit misleading. Why would the size of the eventlog be relevant. What's more important is to minimize the TCB also during the boot flow. The verifier checks the integrity of the log by doing the "replay" to re-compute the RTMR values and compare the final value what is in the quote. The replay is not any harder if there is 10 or 100 entries.  If the replay is OK, the individual log entries can be trusted and appraised independently (without having to check them all).

Reply:
there is a correlation between the event  log size and the TCB, yes, the whole idea is to reduce the TCB, we did not mean that having a big event log will make computation harder but surely, it will make the understanding harder, when you take a look at the event log with traditional boot on Ubuntu 24.10, there is approximately 120 events, it is hard to know which component measures what and the whole "TCB" has been properly measured